app/src/org/storymaker/app/HomeActivity.java

```diff
@@ -87,6 +87,15 @@
     public void onCreate(Bundle savedInstanceState) {
     	super.onCreate(savedInstanceState);
 
+        // set title bar as a reminder if test server is specified
```

Comment:
Why did you put this in HomeActivty vs BaseActivity?

https://github.com/mnbogner/storymaker/blob/base_050715/app/src/org/storymaker/app/BaseActivity.java#L207